### ISSUE TITLE
No OpKernel was registered to support Op TRTEnginOp used by {{node_TRTEngineOp_4}}

### ISSUE THREAD

**Issue by Niloofar-bgh** [ardianumam/Tensorflow-TensorRT#33]
Running 
> 2_inference_using_TensorRT-model.ipynb 

gives the below error, and I have:

- The TensorFlow-GPU 1.15.0
- The Nvidia RTX 2070

Any idea what could be the reason?!

![image](https://user-images.githubusercontent.com/22964938/110042108-44612580-7d13-11eb-95b1-269c85b6fbeb.png)

**Comment by anantgupta129** [comment]
@Niloofar-bgh  I am facing same error did you find any solution to this?

**Comment by Niloofar-bgh** [comment]
> @Niloofar-bgh I am facing same error did you find any solution to this?

Not really; I was trying to run the code on my personal computer, I wasn't successful, then I switched to using Google Colab, which I successfully executed the code! My assumption for the issue is the version incompatibility of libraries and dependencies! However, if you found a solution to the problem, please let me know.
